src/platform_impl/windows/event_loop.rs

```diff
@@ -127,6 +134,10 @@ pub(crate) enum ProcResult {
 pub struct EventLoop {
     window_target: ActiveEventLoop,
     msg_hook: Option<Box<dyn FnMut(*const c_void) -> bool + 'static>>,
+    // It is a timer used on timed waits.
+    // It is created lazily in case if we have `ControlFlow::WaitUntil`.
+    // Keep it as a field to avoid recreating it on every `ControlFlow::WaitUntil`.
+    high_precision_timer: Option<HANDLE>,
```

Comment:
Can we use [`OwnedHandle`](https://doc.rust-lang.org/std/os/windows/io/struct.OwnedHandle.html) for this instead to simplify cleanup?

Reply:
Ok

Reply:
Honestly, it looks much messier as result because types used in `std` and in `windows-sys` crate are not compatible.
Btw, why use `windows-sys` crate instead of `windows` crate? `windows` crate does all error checking itself and has more ergonomic interface overall.